config/jobs/kubernetes/sig-testing/bazel-build-test.yaml

```diff
@@ -216,9 +216,9 @@ periodics:
             --config=ci \
             "--build=//... -//vendor/..." \
             --release=//build/release-tars \
-            --gcs=gs://kubernetes-release-dev/ci \
+            --gcs=gs://kubernetes-release-dev/ci-periodic \
```

Comment:
do we need a periodic and a postsubmit? maybe we should just remove one of these.

Reply:
Now that we have rerun we probably don't need a periodic auto-correct flakes.